modules/components/pages/outputs/redpanda_migrator_bundle.adoc

```diff
@@ -40,7 +41,14 @@ The xref:components:outputs/redpanda_migrator.adoc[`redpanda_migrator` output] c
 
 The xref:components:outputs/schema_registry.adoc[`schema_registry` output] configuration. The `subject` field must be left empty.
 
-
 *Type*: `object`
 
+=== `translate_schema_ids`
+
+When set to `true`, this field automatically translates the schema ID in each message to match the corresponding schema in the destination schema registry. The updated message is then written to the destination schema registry.
```

Comment:
This one is a bit more complicated, since it enables / disables both `redpanda_migrator.translate_schema_ids` and `schema_registry.translate_ids`. Maybe the best thing to do is to link to each of those fields? The schema with a translated ID will be written to the destination schema registry while the message with a translated schema ID (the schema ID is embedded in the first 5 bytes of the message) will be written to the destination broker.